File: beecrowd/1/1046/1046.c
#include <stdio.h>

void printar(int horas) {
    printf("O JOGO DUROU %d HORA(S)\n", horas);
}

int main(void) {
    int inicial, final;
    scanf("%d %d", &inicial, &final);
    if (inicial < final) {
        printar(final - inicial);
    } else if(inicial > final) {
        printar((24 - inicial) + final);
    } else {
        printar(24);
    }
    return 0;
}

File: beecrowd/1/1046/1046.clj
(let [hora-inicial (read)
      hora-final (read)
      calcular-tempo (fn [horas] (printf "O JOGO DUROU %d HORA(S)\n" horas))]
  (cond
    (< hora-inicial hora-final) (calcular-tempo (- hora-final hora-inicial))
    (> hora-inicial hora-final) (calcular-tempo (+ (- 24 hora-inicial) hora-final))
    (= hora-inicial hora-final) (calcular-tempo 24)))
